### PR TITLE
nx-btn-bar alignment - RSC-415

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-415

Causing: https://issues.sonatype.org/browse/CLM-17541

### CODE REVIEW

**Comment by glenhunter** [comment]
Pagination buttons are no longer centred. 
Screen Shot 2020-12-15 at 10 41 37 AM

File: lib/src/base-styles/_nx-btn-bar.scss [hunk 1]
```diff
@@ -8,6 +8,8 @@
 .nx-btn-bar {
   @include container-horizontal;
 
+  align-items: first baseline;
```
Comment (glenhunter) [inline lib/src/base-styles/_nx-btn-bar.scss:11]: I'd prefer `center` since it's clearer in its effect/intent

**Comment by rpokorny** [comment]
> Pagination buttons are no longer centred.

065aac57

File: lib/src/base-styles/_nx-btn-bar.scss (hunk 1 above)
Comment (rpokorny) [reply lib/src/base-styles/_nx-btn-bar.scss]: > I'd prefer center since it's clearer in its effect/intent

8d20874

**Review by glenhunter — APPROVED** [review]

**Pull request merged**